libkbfs/key_bundle_v3.go

```diff
@@ -214,6 +253,32 @@ func DeserializeTLFWriterKeyBundleV3(codec kbfscodec.Codec, path string) (
 	return wkb, nil
 }
 
+func tlfEphemeralPublicKeysSize(keys kbfscrypto.TLFEphemeralPublicKeys) int {
+	pkSize := int(unsafe.Sizeof(kbfscrypto.TLFEphemeralPublicKey{}))
```

Comment:
From https://golang.org/pkg/unsafe/:

> Package unsafe contains operations that step around the type safety of Go programs.
> 
> Packages that import unsafe may be non-portable and are not protected by the Go 1 compatibility guidelines.

It doesn't seem like a good thing to have in production code unless absolutely necessary.